x-pack/plugins/actions/server/plugin.ts

```diff
@@ -645,7 +645,6 @@ export class ActionsPlugin implements Plugin<PluginSetupContract, PluginStartCon
       request,
       authorizationMode,
       authorization: this.security?.authz,
-      authentication: this.security?.authc,
```

Comment:
The `authentication` property was unused and used the deprecated access method to the APIs